### PR TITLE
enh: move Delete button to event settings page

### LINKED ISSUE #3997: Delete Event should be under settings page
We already have an Event Settings Sub-tab in the Events View. 

It would be better to move the functionality of Delete Event from Nav-Bar to the Event Settings Page with a proper description like loss of data, sessions, speakers info.

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3997 

## Desktop View (Latest)

Screenshot from 2020-02-09 06-13-57

## Mobile View (Latest)

Screenshot from 2020-02-09 06-13-47

## Working GIF

captured (2)

### CODE REVIEW

File: app/controllers/events/view/settings.js [hunk 1]
```diff
@@ -3,6 +3,34 @@ import { action } from '@ember/object';
 
 export default class extends Controller {
 
+  @action
+  openDeleteEventModal(id, name) {
+    this.setProperties({
+      'isEventDeleteModalOpen' : true,
+      'confirmEventName'       : '',
+      'eventId'                : id,
```
Comment (vishakhanihore) [inline app/controllers/events/view/settings.js]: Pass eventId and Name from the model, why this unnecessary code

**Comment by kushthedude** [comment]
Leave the template changes, Just move the functionality in the settings section.

**Comment by snitin315** [comment]
@kushthedude Done :+1:

File: app/controllers/events/view/settings.js [hunk 2]
```diff
@@ -3,6 +3,34 @@ import { action } from '@ember/object';
 
 export default class extends Controller {
 
+  @action
+  openDeleteEventModal(id, name) {
+    this.setProperties({
+      'isEventDeleteModalOpen' : true,
```
Comment (kushthedude) [inline app/controllers/events/view/settings.js]: Why setting for properties? Revert the duplication

Comment (snitin315) [reply app/controllers/events/view/settings.js]: @kushthedude done :+1:

**Comment by kushthedude** [comment]
Fix ci

**Comment by snitin315** [comment]
@kushthedude CI check is already passed.

**Pull request merged**